### PR TITLE
Add toggle for all bookmarks

### PR DESCRIPTION
**Pull request by stschuessler**
Created a code block which allows to toggle all bookmarks.

Therefore I used the querySelectorAll method and assigned this List to my switchButtonArray variable. 
I used the forEach method which allows me to go through each element of this list.

### CODE REVIEW

File: javascript.js [hunk 1]
```diff
@@ -63,8 +63,22 @@ const switchButton = document.querySelector('.button__bookmark')
...
-  if (switchButton.classList.contains('button__bookmark--clicked')) {
-    switchButton.classList.remove('button__bookmark--clicked')
-  } else switchButton.classList.add('button__bookmark--clicked')
+// switchButton.addEventListener('click', () => {
+//   if (switchButton.classList.contains('button__bookmark--clicked')) {
+//     switchButton.classList.remove('button__bookmark--clicked')
+//   } else switchButton.classList.add('button__bookmark--clicked')
+// })
+
+/* Switch bookmark Button für alle bookmarks */
```
Comment (macfrei) [inline javascript.js:72]: Please remove code which is commented out! 🙂 
Also, I LOVE your PullRequest description!

File: javascript.js [hunk 2]
```diff
@@ -63,8 +63,22 @@ const switchButton = document.querySelector('.button__bookmark')
...
+
+const switchButtonArray = document.querySelectorAll('.button__bookmark')
+
+switchButtonArray.forEach(switchButton => {
+  switchButton.addEventListener('click', () => {
+    if (switchButton.classList.contains('button__bookmark--clicked')) {
+      switchButton.classList.remove('button__bookmark--clicked')
+    } else {
+      switchButton.classList.add('button__bookmark--clicked')
+    }
```
Comment (macfrei) [inline javascript.js:82]: How about `switchButton.classList.toggle('button__bookmark--clicked')`? 🙂

File: javascript.js (hunk 1 above)
Comment (stschuessler) [reply javascript.js:72]: Thanks! 🙂
And I will remove the mentioned code :wink:

File: javascript.js (hunk 2 above)
Comment (stschuessler) [reply javascript.js:82]: Yes, that's the shorter version of the if...else statement? I think, I used it, when I wrote the code in the early beginning. But I also wanted to see, if I would figure out the if...else version ^^

Comment (macfrei) [reply javascript.js:82]: Ah I see! 🙂

**Pull request merged**